importers/src/bible/osis/functions/sections.functions.ts

```diff
@@ -63,10 +63,10 @@ export const startNewSection = (context: ParserContext, elementType: OsisSection
 };
 
 export const getCurrentSection = (context: ParserContext) => {
-    return context.contentContainerStack.find((container) => {
-        if (container.type === 'section') {
-            return true;
+    for (let i = context.contentContainerStack.length - 1; i >= 0; i--) {
```

Comment:
can you add a quick test case to describe when this comes into play?

does this imply that there could be sections within sections?

I think the change is fine, just want to protect against regressions and document behavior, with test case

Reply:
yes, there can be sections several levels deep. i added a test.